package.json

```diff
@@ -16,105 +16,106 @@
-    "file-loader": "0.11.2",
-    "flood-ui-kit": "^0.1.7",
-    "flux": "^3.1.0",
-    "fs-extra": "^4.0.1",
+    "envify": "^4.1.0",
+    "events": "^2.0.0",
+    "express": "^4.16.3",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "feedsub": "^0.5.2",
+    "file-loader": "1.1.11",
+    "flood-ui-kit": "^0.1.8",
+    "flux": "^3.1.3",
+    "fs-extra": "^5.0.0",
     "geoip-country-only": "^1.1.8",
-    "html-webpack-plugin": "^2.30.1",
+    "html-webpack-plugin": "^3.1.0",
     "inuit-page": "^0.2.1",
     "inuit-reset": "^0.1.1",
-    "joi": "^13.0.2",
-    "jsonwebtoken": "^7.0.1",
+    "joi": "^13.1.2",
+    "jsonwebtoken": "^8.2.0",
     "jsx-loader": "^0.13.2",
-    "lodash": "^4.3.0",
+    "lodash": "^4.17.5",
     "mkdirp": "^0.5.1",
-    "moment": "^2.14.1",
-    "morgan": "^1.5.3",
-    "multer": "^1.1.0",
+    "moment": "^2.21.0",
+    "morgan": "^1.9.0",
+    "multer": "^1.3.0",
     "mv": "^2.1.1",
-    "nedb": "^1.7.2",
-    "node-sass": "^4.6.1",
+    "nedb": "^1.8.0",
+    "node-sass": "^4.8.3",
     "object-assign": "4.1.1",
     "ospath": "^1.2.2",
-    "passport": "^0.3.2",
-    "passport-jwt": "^2.1.0",
-    "postcss-flexbugs-fixes": "^3.2.0",
-    "postcss-loader": "2.0.6",
+    "passport": "^0.4.0",
+    "passport-jwt": "^4.0.0",
+    "postcss-flexbugs-fixes": "^3.3.0",
+    "postcss-loader": "2.1.3",
     "promise": "^8.0.1",
-    "pug": "^2.0.0-rc.3",
-    "react": "^15.6.1",
-    "react-addons-create-fragment": "^15.6.0",
-    "react-addons-css-transition-group": "^15.6.0",
-    "react-custom-scrollbars": "^4.0.0",
-    "react-dev-utils": "^3.0.2",
-    "react-dnd": "^2.2.3",
-    "react-dnd-html5-backend": "^2.2.3",
+    "pug": "^2.0.3",
+    "react": "^15.6.2",
+    "react-addons-create-fragment": "^15.6.2",
+    "react-addons-css-transition-group": "^15.6.2",
+    "react-custom-scrollbars": "^4.2.1",
+    "react-dev-utils": "^5.0.0",
+    "react-dnd": "^2.6.0",
+    "react-dnd-html5-backend": "^2.6.0",
     "react-document-title": "^2.0.3",
-    "react-dom": "^15.6.1",
-    "react-dropzone": "^3.4.0",
-    "react-error-overlay": "^1.0.9",
-    "react-intl": "^2.1.3",
-    "react-router": "^3.0.0",
-    "react-transition-group": "^2.2.0",
-    "reset-css": "^2.2.1",
+    "react-dom": "^15.6.2",
+    "react-dropzone": "^4.2.9",
+    "react-error-overlay": "^4.0.0",
+    "react-intl": "^2.4.0",
+    "react-router": "^3.2.1",
+    "react-transition-group": "^2.3.0",
+    "reset-css": "^3.0.0",
     "ress": "^1.2.2",
-    "rimraf": "^2.6.1",
-    "run-series": "^1.1.4",
-    "sass-loader": "^6.0.6",
+    "rimraf": "^2.6.2",
+    "run-series": "^1.1.6",
+    "sass-loader": "^6.0.7",
     "spdy": "^3.4.7",
-    "style-loader": "0.18.2",
-    "tar-stream": "^1.5.4",
-    "transform-loader": "^0.2.3",
-    "url-loader": "0.5.9",
+    "style-loader": "0.20.3",
+    "tar-stream": "^1.5.5",
+    "transform-loader": "^0.2.4",
+    "url-loader": "1.0.1",
     "webpack": "^3.4.1",
-    "xmlrpc": "^1.3.0"
+    "ajv": "^6.4.0",
```

Comment:
@GottZ I removed `ajv` and everything compiled as expected in node 8 and 9. Please remove.